NoteFind/Views/NoteCell.m

```diff
@@ -5,13 +5,24 @@
 //  Created by Khloe Wright on 7/8/22.
 //
 
+#import "AppDelegate.h"
+#import <CoreData/CoreData.h>
 #import "NoteCell.h"
 
+
+@interface NoteCell () {
+    AppDelegate *appDelegate;
```

Comment:
We shouldn't keep a reference to the AppDelegate / Context here since you can always access it from `[UIApplication sharedApplication].delegate`.

Additionally this sets up a retain cycle / memory leak since 
AppDelegate retains Feed which retains Note Cells which now retains the AppDelegate

Reply:
An alternative would also to just make both of these instance variables `weak` in order to make sure we aren't holding a strong reference to them

Reply:
okay, since i do everything pertaining offline notes in the downloading, i'll put both variables in there

Reply:
^ didTapDownload